### PR TITLE
chore: update carousel example data to use correct JSS styles

### PR DESCRIPTION
**Pull request by nalogiudice**
Closes [#1493](https://github.com/Microsoft/fast-dna/issues/1493)

# Description
Small updates to the carousel to fix a few bugs:
- fixes incorrect mapping in example data
- fixes incorrect JSS
- streamlines JSS

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-components-styles-msft/src/carousel/index.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
     const white: string = "white";
     const darkGray: string = "#101010";
     const lightGray: string = "rgb(206, 206, 206)";
-    const lightGrayRest: string = "rgba(206, 206, 206, 0.6";
+    const lightGrayRest: string = "rgba(206, 206, 206, 0.6)";
```
Comment (eljefe223) [inline packages/fast-components-styles-msft/src/carousel/index.ts]: I already fixed this yesterday. 
https://github.com/Microsoft/fast-dna/pull/1496

File: packages/fast-components-styles-msft/src/carousel/index.ts [hunk 2]
```diff
@@ -66,15 +66,21 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
...
+                    },
                 },
             },
         },
-        carousel_sequenceIndicator__active: {},
+        carousel_sequenceIndicator__active: {
+            "&::before": {
+                opacity: "1",
+            },
+        },
```
Comment (chrisdholt) [inline packages/fast-components-styles-msft/src/carousel/index.ts]: Can we do this in a different PR? The example data is just a "chore", whereas this is a "fix". Seperating these PR's will keep things clear in the changelog and history.

Comment (nalogiudice) [reply packages/fast-components-styles-msft/src/carousel/index.ts]: Removed style changes from this PR
